### PR TITLE
Making ampath forms work on Vitals section

### PR DESCRIPTION
**Pull request by MiguelAHPpih**
## Summary

The vitals section has been modified to include the usage of ampath forms as an option(implementer tools).
This was tested using "POC Sample Form 1".

## Related Issue
When the form is being loaded and there isn't an active visit. It will crash with no logs in the console.
This is adressed in this ticket:
https://issues.openmrs.org/browse/O3-1243

### CODE REVIEW

**Comment by brandones** [comment]
Hi @MiguelAHPpih , can you please attach a screenshot, gif, or video?

**Comment by MiguelAHPpih** [comment]
@brandones 
Vitals-AmpathForms

File: packages/esm-patient-vitals-app/src/index.ts [hunk 1]
```diff
@@ -1,76 +1,76 @@
...
-            default: 'Record Vitals and Biometrics',
-          },
-        },
-      },
-    ],
-  };
-}
-
-export { backendDependencies, importTranslation, setupOpenMRS };
+import { defineConfigSchema, fhirBaseUrl, getAsyncLifecycle, messageOmrsServiceWorker } from '@openmrs/esm-framework';
```
Comment (brandones) [inline packages/esm-patient-vitals-app/src/index.ts:1]: What happened to this file? Seems like some strange whitespace change happened. Maybe `yarn prettier` or just `git checkout origin/master packages/esm-patient-vitals-app/src/index.ts`?

Comment (MiguelAHPpih) [reply packages/esm-patient-vitals-app/src/index.ts:1]: I remember doing some tests with console.log, but only that. It could be prettier

Comment (brandones) [reply packages/esm-patient-vitals-app/src/index.ts:1]: Please run `git checkout origin/master packages/esm-patient-vitals-app/src/index.ts` to revert these changes, and then `yarn prettier`. Commit whatever results.

Comment (MiguelAHPpih) [reply packages/esm-patient-vitals-app/src/index.ts]: Done

**Review by brandones — APPROVED** [review]

**Comment by brandones** [comment]
@denniskigen Looks like the build step is OOMing. This also recently happened on master: https://github.com/openmrs/openmrs-esm-patient-chart/actions/runs/2238756863 . Maybe we need to limit the number of things that can run in parallel?

**Comment by denniskigen** [comment]
> @denniskigen Looks like the build step is OOMing. This also recently happened on master: https://github.com/openmrs/openmrs-esm-patient-chart/actions/runs/2238756863 . Maybe we need to limit the number of things that can run in parallel?

Perhaps. I wonder though whether the build step has to be rerun inside of this `pre-release` job? Is it possible to reuse the pre-built artefacts from the `build` job?

**Comment by brandones** [comment]
@denniskigen That would be great. I'm inclined to merge this for now since I don't think the problem has anything to do with @MiguelAHPpih 's code.

**Pull request merged**